actors/evm/src/interpreter/memory.rs

```diff
@@ -3,6 +3,9 @@ use {
     derive_more::{Deref, DerefMut},
 };
 
+use crate::EVM_WORD_SIZE;
+
+// REMOVEME actuall wasm page is 64Kib no?
```

Comment:
?

Reply:
Yes. And yeah, we should probably just use that (in a new PR).